scripts/draft-translate.py

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash python3
```

Comment:
```suggestion
#!/usr/bin/env python
```

Reply:
@sftim @nasa9084 
Thank you for your suggestion.
I have changed to `#!/usr/bin/env python3`